tests/software/cnvkit/main.nf

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { CNVKIT } from '../../../../software/cnvkit/main.nf' addParams( options: [ publish_dir:'test_with_normal' ] )
+include { CNVKIT } from '../../../../software/cnvkit/main.nf' addParams( options: [ publish_dir:'test_without_normal' ] )

```

Comment:
You can't include twice the same module if they have the same name.
You should be able to use `as` to rename the module, similar to what we're doing in the TABIX_TABIX module for example:
https://github.com/MaxUlysse/nf-core_modules/blob/01c0d335ca57b1b9298a7acd643116917953804a/tests/software/tabix/tabix/main.nf#L5-L7

Reply:
ok. Thanks

Reply:
Maxime, do I need to add my institute's profile in the config file to use or test this module at my work place?

Reply:
The whole explanation for CI is here: https://github.com/nf-core/modules#testing
You should be able to test on your local computer, no need to do that on your institute HPC.
But it's actually a good question @drpatelh @KevinMenden Is it possible to test locally using an institute profile?

Reply:
Sorry I overlooked that mention somehow .... not really sure about running locally with a different config profile.
There shouldn't be any extra configs for the module necessary, I think as long as you have one for the pipeline (which you'll need anyway), you're fine (not really adding much knowledge here I now 😄 )

Reply:
thanks Kevin